sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -745,43 +745,48 @@ case class WholeStageCodegenExec(child: SparkPlan)(val codegenStageId: Int)
 
     val durationMs = longMetric("pipelineTime")
 
+    val ignoreCorruptRecord: Boolean = conf.ignoreCorruptRecord
```

Comment:
It's a hive specific conf. can we avoid fixing the sql execution core?

Reply:
Thanks for reviewing. In my case, we need it to catch [java.lang.ArrayIndexOutOfBoundsException],[java.lang.NullPointerException], when we queried sequence file table(mixed schema in one table) .

It confused me too, but this solution worked